Sources/Commands/SwiftTestTool.swift

```diff
@@ -173,10 +182,10 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
             to: { $0.shouldRunInParallel = $1 })
 
         binder.bind(
-            option: parser.add(option: "--specifier", shortName: "-s", kind: String.self,
-                usage: "Run a specific test class or method, Format: <test-target>.<test-case> or " +
+            option: parser.add(option: "--filter", kind: String.self,
+                usage: "Run test cases matching regular expression, Format: <test-target>.<test-case> or " +
```

Comment:
We will need to keep the `--specifier` for now and show a deprecation notice. See `fetch` in package tool for example.